tests/e2e/Shared/products.js

```diff
@@ -0,0 +1,11 @@
+const simple = {
```

Comment:
This is a subset of products that I need to feed from the ones I create in the build, still to do...